code/modules/food_and_drinks/food/snacks_bread.dm

```diff
@@ -261,7 +261,7 @@
 	. = ..()
 
 /obj/item/reagent_containers/food/snacks/deepfryholder/On_Consume(mob/living/eater)
-	if(contents)
+	if(contents && !reagents.total_volume)
```

Comment:
Does the `contents` check do anything? Won't it always be at least an empty list, and thus truthy?

Reply:
ah yeah, i dont think anyone even noticed that this was the case due to the fact no one uses normal deep frying

fixed